docs/background_jobs.md

```diff
@@ -0,0 +1,35 @@
+# Background jobs
+
+[Sidekiq](https://github.com/sidekiq/sidekiq) is the background job processor we use that handles long-running tasks, such as sending emails or processing images, asynchronously.
+
+## Configuration
+
+For sidekiq to work and process jobs, you need to setup two things:
+
+1. A worker `bundle exec sidekiq` needs to be added on your cloud platform and running. This command should be run on your local as well.
```

Comment:
```suggestion
1. A worker `bundle exec sidekiq` needs to run for the jobs to be processed. 
```

Reply:
mention how if they are running the server with `bin/dev` then the process run automatically for local development.